### ISSUE TITLE
[Enhancement]: Поддержка многоязычного интерфейса

### ISSUE THREAD

**Issue by Bondualf** [0x90d/videoduplicatefinder#497]
## Environment
 - OS: [e.g. Windows / Linux / MacOS ]

## Describe desired feature
Hello. It would be great if I could use the program in Ukrainian and Russian.

**Comment by 0x90d** [comment]
I agree but currently VDF does not support any localization. And I dont know how to maintain and validate user translations.

**Comment by ssuyoungwu** [comment]
Well,i think i understand your concern. I believe you can enlist the help of others to translate into different languages. You could put out a translation request in your "README.md" file for speakers of various languages, then use ChatGPT's translation to verify the results, ensuring they're generally accurate and don't carry unintended meanings.

In fact, I have already used Visual Studio 2022 and ChatGPT to translate your project into Chinese. I found that ChatGPT's translation results are not entirely accurate, and they need to be polished by someone who is proficient in the language.

And you, my friend, will have an increased workload. You'll need to modify the program to support multiple languages and take on the task of verifying the final translation results, which will make your job tougher. 

The God confounded the languages of the people at the Tower of Babel, preventing them from communicating with each other. But now, with the help of others and ChatGPT, we can make your program more functional and excellent. This, indeed, embodies the spirit and charm of open source, doesn't it?

![PixPin_2024-04-27_22-06-01](https://github.com/0x90d/videoduplicatefinder/assets/45707612/e61285e5-ebd7-4699-a37f-71ac2b816394)
![PixPin_2024-04-27_22-06-09](https://github.com/0x90d/videoduplicatefinder/assets/45707612/297717b1-6bcf-47cf-8720-10b194685b1a)
